### DISCUSSION TITLE
communication structure for API address book

### DISCUSSION THREAD

**Discussion by SoftwareMagicIT** [rustdesk/rustdesk#1154] [General]
Hello, I discovered your software, I like it very much, I would like to contribute and I would also like to use it for my work, but I need to implement the address book. For this I am willing to share the code that I will write, I have all this month available. But I would need the structure, preferably in json so that I can implement the address server with its database. This is enough for me then I will share the source of the API call.

**Comment by rustdesk** [comment]
https://github.com/boonkerz/rustdesk-addressbook

**Reply by SoftwareMagicIT** [reply]
A thousand thanks. I didn't think it was already there. I take this opportunity here for two more small questions. The first is because in the version compiled by me the audio does not work under Windows and the second is: can you integrate the dll sciter, until it passes under flutter and have it extracted if not found, when starting the application? I saw that by placing the dll in \ windows \ system32 the application works and since it has administrator permissions, when installing or starting, you could check if the file exists and in case you could download it there. Sorry my bad english, i am using google translate

**Reply by rustdesk** [reply]
PR welcome
